ReleaseProcessScript/tools/ReleaseProcessScript.psm1

```diff
@@ -112,7 +112,7 @@ function Push-Remote-Repositories ()
     [switch]$WithTags
   )
 
-  Push-To-Repos $Branchname $WithTags
+  Push-To-Repos $Branchname
```

Comment:
I thought it would be fitting to push no tags for the time being (until https://re-motion.atlassian.net/browse/RMSRCBUILD-152) such that the entry point still "works" (with the tag(s) being pushed manually) - especially since the $WithTags switch (formerly second parameter of `Push-To-Repos`) does not exist anymore. One would get an error that there is no tag/refspec `True`/`False` when trying to push.

Or should I better leave it untouched?

Reply:
If you remove the parameter, you also need to remove the parameter's help text. But I have an entirely different question: After your changes, is the ReleaseProcessScript in a "consistent state" now? Because to me, it almost looks like it would't be, because we have changed the infrastructure without changing the entry point because we "don't quite know what to do exactly". If anything, we probably should look at [RMSRCBUILD-152](https://re-motion.atlassian.net/browse/RMSRCBUILD-152) first, before merging this PR and actually make a final design decision (or have them both ready so we can merge them in quick succession). What does @MichaelKetting think?

Reply:
You are right regarding the parameter's help text. However, in the current form, the parameter is still there, but unused (which might be even worse) - that's why I temporarily kept the help text since I was unsure how to proceed with this entrypoint.

I created [RMSRCBUILD-152](https://re-motion.atlassian.net/browse/RMSRCBUILD-152) after a conversation with @MichaelKetting asking what to do with this entrypoint because we cannot know the tag to be pushed without further introspection (e. g. instead of directly calling `Push-To-Repos` retrieving the tag name based on the branch name or the like). He suggested creating the ticket and giving thought to this issue a bit later on.

His main reasoning was that `Push-Remote-Repositories` is an entrypoint that's not used that often and manually pushing a tag afterwards (or simply knowing not to use this entrypoint for the time being) is not a big burden. On the other hand, it might be quite cumbersome if (many) tags that coincidentally are lying around locally are pushed to the remote and one has to delete them all afterwards. That's why, I think, we do want to merge this PR before starting work on [RMSRCBUILD-152](https://re-motion.atlassian.net/browse/RMSRCBUILD-152).

Reply:
I don't remember if I wanted to drop the tags at this unrelated function. It may have been dicussed but there's not need tp change the behavior of unrelated code in this PR. The Push-to-repos is not used anyway. So long as that's the case, we can fix / drop Push-to-repos separately and keep things clean.

Reply:
I have reverted the changes to this file in order to concentrate on this issue later on separately.